src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -240,103 +239,6 @@ public function hasAuthenticatedBackendUser()
 
 
 	/**
-	 * Add the template output to the cache and add the cache headers
-	 */
-	protected function addToCache()
```

Comment:
We definitely loose a HOOK in here, so this may be a code flow change.
Granted, it should only be used to alter the cache key but might have been used for doing other stuff.
I still am undecided.